sklearn/preprocessing/data.py

```diff
@@ -136,10 +138,18 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
     StandardScaler: Performs scaling to unit variance using the``Transformer`` API
         (e.g. as part of a preprocessing :class:`sklearn.pipeline.Pipeline`).
 
+    References
+    ----------
+    .. [1] Gelman, A. (2008). Scaling regression inputs by dividing by two standard deviations.
+    Statistics in medicine, 27(15), 2865-2873.
     """  # noqa
     X = check_array(X, accept_sparse='csc', copy=copy, ensure_2d=False,
                     estimator='the scale function', dtype=FLOAT_DTYPES,
                     force_all_finite='allow-nan')
+    if with_std:
```

Comment:
`if with_std not in (1, 2, True, False):`

Reply:
E    ValueError: Invalid value for `with_std`: False

Reply:
Done